taca/utils/undetermined.py

```diff
@@ -0,0 +1,164 @@
+import re
+import subprocess
+import gzip
+import glob
+import os
+import logging
+import taca.flowcell_parser.classes as cl
+from taca.utils.config import CONFIG
+
+logger=logging.getLogger(__name__)
+#dmux_folder=CONFIG['analysis']['bcl2fastq']['options']['output_dir']
```

Comment:
well, I'd like to keep it actually, but it did not work with the standalone test, so I'm keeping it there until I can run a proper test